src/modules/esl-share/actions/mail-action.ts

```diff
@@ -0,0 +1,15 @@
+import {ESLShareUrlGenericAction} from './url-generic-action';
+
+@ESLShareUrlGenericAction.register
+export class ESLShareMailAction extends ESLShareUrlGenericAction {
+  public static readonly is: string = 'mail';
+
+  public do(): void {
```

Comment:
It doesn't look like Mail action have anything related with mail except name...

Reply:
It was renamed to external action